rust/operator-binary/src/airflow_db_controller.rs

```diff
@@ -188,7 +188,7 @@ fn build_init_job(airflow_db: &AirflowDB) -> Result<Job> {
 
     let container = ContainerBuilder::new("airflow-init-db")
         .image(format!(
-            "docker.stackable.tech/stackable/airflow:{}-stackable0",
```

Comment:
One hard-coded reference to `stackable0` still exists in `airflow_controller.rs` that should be removed in a similar way:
https://github.com/stackabletech/airflow-operator/blob/feat/adr18/rust/operator-binary/src/airflow_controller.rs#L288

Reply:
Removed